### PR TITLE
Newtoki

### LINKED ISSUE #2475: [Newtoki] Connector not working
**Did you read [the troubleshooting guide](https://hakuneko.download/docs/troubleshoot/)**
Yes

**Is the website of the connector working properly / are you able to see the manga within your browser**
Yes

**Describe the bug**
When trying to download from newtoki##.com (newtoki82.com at the moment), no files are actually downloaded. The connector works and the chapter list gets fetched. When downloading, it creates the title folder and the chapter folders but they're empty. No errors appear even after finishing.

**To Reproduce**
1. Newtoki
2. 오늘도 해맑음
3. All chapters
4. No relevant error messages

**Screenshots**
If complex to describe, add screenshots to help explain your problem.

**HakuNeko (please complete the following information):**
 - Version 6.17

### PR DESCRIPTION
**Pull request by Robonau**
moved some manatoki stuff too
also newtoki = manatoki, like exactly
resolves #2475

### CODE REVIEW

**Comment by Robonau** [comment]
clipboard methods don't work on either of these connectors 

for https://newtoki83.com/bbs//board.php?bo_table=webtoon&wr_id=459552

note: it doesn't even get to _getMangaFromURI so i have no idea

File: src/web/mjs/connectors/NewToki.mjs [hunk 1]
```diff
@@ -1,99 +1,44 @@
...
 
-export default class NewToki extends Connector {
+export default class NewToki extends GnuBoard5BootstrapBasic2 {
 
     constructor() {
         super();
         super.id = 'newtoki';
         super.label = 'NewToki';
         this.tags = [ 'manga', 'webtoon', 'korean' ];
         this.url = 'https://newtoki.com';
```
Comment (27136773) [inline src/web/mjs/connectors/NewToki.mjs]: URL cannot be reached, currently it is https://newtoki83.com
Consider using the same getter/setter mechanism to store the URL during `initializeConnector` in settings as it is done in [CopyToon](https://github.com/manga-download/hakuneko/blob/master/src/web/mjs/connectors/CopyToon.mjs).

Maybe also fix the typo `regularly` in [CopyToon.mjs#L21](https://github.com/manga-download/hakuneko/blob/master/src/web/mjs/connectors/CopyToon.mjs#L21)

File: src/web/mjs/connectors/NewToki.mjs [hunk 2]
```diff
@@ -1,99 +1,44 @@
...
+
         this.path = [ '/webtoon', '/comic' ];
+        this.queryMangasPageCount = 'div.list-page ul.pagination li:last-child a';
+        this.queryMangas = 'ul#webtoon-list-all li div.img-item div.in-lable a';
+        this.queryManga = 'meta[name="subject"]';
+        this.queryChapters = 'div.serial-list li.list-item div.wr-subject a';
+        this.scriptPages = `
+        new Promise(resolve => {
+            const queryPages = document.querySelectorAll('div.view-padding div > img').length === 0 ? 'div.view-padding div > p:not([class]) img' : 'div.view-padding div > img';
```
Comment (27136773) [inline src/web/mjs/connectors/NewToki.mjs]: May improve CSS query
```javascript
const images = [...document.querySelectorAll('div.view-padding div > img, div.view-padding div > p:not([class]) img')];
resolve(images.map(image => ...));
```

File: src/web/mjs/connectors/NewToki.mjs [hunk 3]
```diff
@@ -1,99 +1,44 @@
...
+            resolve([...document.querySelectorAll(queryPages)].map(image => JSON.stringify(image.dataset).match(/"\\S{11}":"(.*)"/)[1]));
+        });
+        `;
     }
 
     canHandleURI(uri) {
         return /https?:\/\/newtoki\d*.com/.test(uri.origin);
     }
 
-    async _initializeConnector() {
```
Comment (27136773) [inline src/web/mjs/connectors/NewToki.mjs]: May keep initialize to update URL like in CopyToon

File: src/web/mjs/connectors/NewToki.mjs (hunk 2 above)
Comment (Robonau) [reply src/web/mjs/connectors/NewToki.mjs:19]: the image is not always inside of p:not([class])

Comment (Robonau) [reply src/web/mjs/connectors/NewToki.mjs:19]: that's why i have the tyranny in the first place

**Comment by Robonau** [comment]
side note manatoki.com redirects to newtoki83.com and not to manatoki83.net
(manatoki.net still redirects to manatoki83.net)

**Review by 27136773 — APPROVED** [review]
Looks good

**Pull request merged**